app/views/static_pages/community_expectations.html.erb

```diff
@@ -1,11 +1,87 @@
 <%= title('Community Expectations') %>
-<% filename = 'app/views/static_pages/dashboard_steps/community_expectations.md' %>
+<div class="gradient py-8">
```

Comment:
We have a page-container that be used here. It will give us consistent spacing between navbar/footer and content for all pages.

```suggestion
<div class="page-container">
```